src/Test/WebDriver/Commands/Wait.hs

```diff
@@ -131,8 +130,7 @@ wait' handler waitAmnt t wd = waitLoop =<< liftBase getCurrentTime
         retry why = do
           now <- liftBase getCurrentTime
           if diffUTCTime now startTime >= timeout
-            then
-              failedCommand Timeout $ "wait': explicit wait timed out (" ++ why ++ ")."
+            then throwIO why
```

Comment:
Note that this obscures the fact that a timeout occurs. I think it's worth it though--I'd rather have the nicely printed explanation of the underlying exception than a messy string with `wait': explicit wait timed out (...)`.

I haven't explored this but maybe there's another standard exception that this could be wrapped in, that would signify a timeout occurred without breaking pretty-printing...